JitsiConference.js

```diff
@@ -116,6 +117,19 @@ JitsiConference.prototype._init = function (options) {
     // Always add listeners because on reload we are executing leave and the
     // listeners are removed from statistics module.
     this.eventManager.setupStatisticsListeners();
+
+    if (this.options.config.enableTalkWhileMuted) {
+        this.talkMutedDetection
+            = new TalkMutedDetection(() => {
+                this.eventEmitter.emit(JitsiConferenceEvents.TALK_WHILE_MUTED);
```

Comment:
Why is `TALK_WHILE_MUTED` an event of `JitsiConference`? Reading `TalkMutedDetection`, it looks to me like it's mostly about the local audio track so I'm wondering why `TALK_WHILE_MUTED` is not an event of the local audio track.